### DISCUSSION TITLE
在OpenUPM的web上能显示，为啥Unity里面却没有？

### DISCUSSION THREAD

**Discussion by ClaineLe** [openupm/openupm#2432] [Q&A]
如题。
包名：com.snake.framework.fork-fairygui

**Comment by favoyang** [comment]
Please provide the below information

- Unity version
- Packages/manifest.json file

Refs
- https://openupm.com/docs/getting-started.html#installing-openupm-cli
- (Chinese docs: https://openupm.cn/zh/docs/getting-started.html#%E5%AE%89%E8%A3%85openupm-cli)
